projects/@shared/lib/validator/extensions/image.js

```diff
@@ -28,7 +28,7 @@ const image = joi => ({
         },
       ],
       validate(value, helpers, args) {
-        if (value.width < args.width) {
+        if (value?.width && value.width < args.width) {
```

Comment:
`value?.width && value.width < args.width` -> `value?.width < args.width` `undefined` is altijd `false`, dus dit kan in 1 keer.